scenic/projects/robust_segvit/datasets/fishyscapes.py

```diff
@@ -0,0 +1,207 @@
+"""Fishyscapes Lost & Found Validation Dataset.
+
+Edited from
+https://github.com/hermannsblum/bdl-benchmark/blob/master/bdlb/fishyscapes/fishyscapes_tfds.py
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
```

Comment:
nit: remove these `from __future__` lines since we only support python3 now

Reply:
Done